javascript/extractor/src/com/semmle/js/extractor/YAMLExtractor.java

```diff
@@ -216,33 +215,30 @@ private String getTag(String explicitTag, NodeId kind, String value, boolean imp
     return explicitTag;
   }
 
+  private static boolean isNewLine(int codePoint) {
+    switch (codePoint) {
+      case '\n':
+      case '\r':
```

Comment:
(I suppose it is fine in practice for our toStrings to treat spuriously placed `\r` characters as ordinary newlines.)